qfieldsync/core/cloud_api.py

```diff
@@ -69,7 +69,10 @@ def from_reply(reply: QNetworkReply) -> Optional[CloudException]:
                 message = resp['detail']
         except:
             if payload:
-                message = payload[:500] + '...'
+                message = payload[:500] 
+
+                if len(payload) > 500:
+                    message += '…'
```

Comment:
Sorry for the missing ellipsis

Reply:
do i miss something here?

Reply:
I think you corrected my three points to a proper ellipsis

Reply:
Yup, I did a separate commit with real elllipsis, instead of three dots.